apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtaksvurderingRoute.kt

```diff
@@ -154,6 +156,10 @@ fun Route.vedtaksvurderingRoute(
 }
 
 fun Route.samordningsvedtakRoute(service: VedtaksvurderingService) {
+    install(AuthorizationPlugin) {
+        roles = setOf("samordning-read")
```

Comment:
Foreslår å flytte AzureGroups frå behandling ut i eit lib (behandling-model, kanskje? litt usikker), og så leggje inn denne også der. Kjens tryggast å ha alle roller samla.

Reply:
P.t. ettersom ingen andre apper har behov for å vite noe om denne rollen så er jeg litt skeptisk til å slå sammen med `AzureGroups`, som jeg opplever som mer funksjonelt orientert enn ren sikkerhet.